examples/multiple-receive/multiple_receive.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+
+type Result record {
+    int a;
+    int b;
+};
+
+public function main() {
+    worker w1 {
+        2 -> w3;
+    }
+
+    worker w2 {
+        runtime:sleep(2);
+        3 -> w3;
+    }
+
+    worker w3 returns json {
+        // The worker waits until both values are received.
+        Result result = <- {a: w1, b: w2};
+        return result.toJson();
+    }
+
```

Comment:
Can we redo this similar to the alternate receive BBE?

w1 and w2 do two fetches. Once both complete w3 does another fetch, sets a field to the record and returns the value?

```ballerina
type Result record {|
    T a;
    T b;
    T c?;
|};
```